mm2src/coins/lp_coins.rs

```diff
@@ -2150,15 +2158,22 @@ impl CoinsContext {
         })))
     }
 
-    pub async fn add_coin(&self, coin: MmCoinEnum) -> Result<(), MmError<CoinIsAlreadyActivatedErr>> {
+    pub async fn add_token(&self, coin: MmCoinEnum) -> Result<(), MmError<CoinIsAlreadyActivatedErr>> {
```

Comment:
No need to change this function's name, `add_coin` is better.

Reply:
At this function we insert this coin ticker into `CoinsContext::platform_coin_tokens` as a token. I think `add_coin` is unclear name.

Reply:
I didn't notice that. You are right, we should keep it as `add_token`.

Reply:
In this case, there's no `abortable_system` attached to `LightningCoin` but only `LightningCoin::platform`